docs/preview/yamlgettingstarted.md

```diff
@@ -30,6 +30,8 @@ YAML builds require the latest agent. When a YAML build is queued, your agent wi
 
 [Triggers](yamlgettingstarted-triggers.md)
 
+[Templates](yamlgettingstarted-templates.md)
```

Comment:
I'm wondering whether we should link to the resources doc from here. I'm thinking that we should, but since resources are interesting on their own, we would need to add links from the resources doc to their usages (templates, eventually checkout, and containers)